2013-09-02 13:45:09 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [1049] Unknown database \'todo\'',
  'enabled' => true,
)
Request URL: /todos
Stack Trace:
#0 /Users/gertpoppe/projects/cakephp/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /Users/gertpoppe/projects/cakephp/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /Users/gertpoppe/projects/cakephp/lib/Cake/Model/Model.php(3288): ConnectionManager::getDataSource('default')
#3 /Users/gertpoppe/projects/cakephp/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /Users/gertpoppe/projects/cakephp/lib/Cake/Model/Model.php(3310): Model->setSource('todos')
#5 /Users/gertpoppe/projects/cakephp/lib/Cake/Model/Model.php(2750): Model->getDataSource()
#6 /Users/gertpoppe/projects/cakephp/lib/Cake/Model/Model.php(2722): Model->_readDataSource('all', Array)
#7 /Users/gertpoppe/projects/cakephp/app/Controller/TodosController.php(5): Model->find('all')
#8 [internal function]: TodosController->index()
#9 /Users/gertpoppe/projects/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(TodosController), Array)
#10 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#11 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(TodosController), Object(CakeRequest), Object(CakeResponse))
#12 /Users/gertpoppe/projects/cakephp/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-09-02 13:50:02 Error: [MissingControllerException] Controller class TestController could not be found.
Exception Attributes: array (
  'class' => 'TestController',
  'plugin' => NULL,
)
Request URL: /test
Stack Trace:
#0 /Users/gertpoppe/projects/cakephp/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-02 13:50:32 Error: Fatal Error (1): Call to a member function find() on a non-object in [/Users/gertpoppe/projects/cakephp/app/Controller/TestController.php, line 5]
2013-09-02 13:50:32 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /test
Stack Trace:
#0 /Users/gertpoppe/projects/cakephp/lib/Cake/Error/ErrorHandler.php(205): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/gertpopp...', 5)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/gertpopp...', 5, Array)
#2 /Users/gertpoppe/projects/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/gertpopp...', 5, Array)
#3 /Users/gertpoppe/projects/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-02 13:55:21 Error: [MissingViewException] View file "/Users/gertpoppe/projects/cakephp/app/View/Test/index.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/gertpoppe/projects/cakephp/app/View/Test/index.ctp',
)
Request URL: /test
Stack Trace:
#0 /Users/gertpoppe/projects/cakephp/lib/Cake/View/View.php(459): View->_getViewFileName(NULL)
#1 /Users/gertpoppe/projects/cakephp/lib/Cake/Controller/Controller.php(952): View->render(NULL, NULL)
#2 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(TestController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/gertpoppe/projects/cakephp/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-09-06 10:32:12 Error: Fatal Error (1): Call to a member function saveField() on a non-object in [/Users/gertpoppe/projects/cakephp/app/Controller/TodosController.php, line 24]
2013-09-06 10:32:12 Error: [FatalErrorException] Call to a member function saveField() on a non-object
Request URL: /todos/resolve/1
Stack Trace:
#0 /Users/gertpoppe/projects/cakephp/lib/Cake/Error/ErrorHandler.php(205): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/gertpopp...', 24)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/gertpopp...', 24, Array)
#2 /Users/gertpoppe/projects/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/gertpopp...', 24, Array)
#3 /Users/gertpoppe/projects/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-06 10:39:47 Error: Fatal Error (4): syntax error, unexpected 'throw' (T_THROW) in [/Users/gertpoppe/projects/cakephp/app/Controller/TodosController.php, line 24]
2013-09-06 10:39:47 Error: [FatalErrorException] syntax error, unexpected 'throw' (T_THROW)
Request URL: /todos
Stack Trace:
#0 /Users/gertpoppe/projects/cakephp/lib/Cake/Error/ErrorHandler.php(205): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/Users/gertpopp...', 24)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/Users/gertpopp...', 24, Array)
#2 /Users/gertpoppe/projects/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/Users/gertpopp...', 24, Array)
#3 /Users/gertpoppe/projects/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-06 10:40:21 Error: [NotFoundException] Could not find your todo
Request URL: /todos/resolve/3
Stack Trace:
#0 [internal function]: TodosController->resolve('3')
#1 /Users/gertpoppe/projects/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(TodosController), Array)
#2 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#3 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(TodosController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/gertpoppe/projects/cakephp/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-09-06 10:42:20 Error: [NotFoundException] Could not find your todo
Request URL: /todos/resolve/4
Stack Trace:
#0 [internal function]: TodosController->resolve('4')
#1 /Users/gertpoppe/projects/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(TodosController), Array)
#2 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#3 /Users/gertpoppe/projects/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(TodosController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/gertpoppe/projects/cakephp/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
